common-services/webapi/__files/product.json

```diff
@@ -1,5 +1,7 @@
 {
   "id": 21762532,
   "url": "http://knotx.io",
-  "label": "Product"
+  "label": "Product",
```

Comment:
Please provide some information in `./README.md` about the target API responses (what is changing) and then explain API Gateway responses (nothing is changed until cached value is valid).

Reply:
done